pandas/tools/util.py

```diff
@@ -61,6 +61,13 @@ def to_numeric(arg, errors='raise'):
         - If 'raise', then invalid parsing will raise an exception
         - If 'coerce', then invalid parsing will be set as NaN
         - If 'ignore', then invalid parsing will return the input
+    downcast : str, numpy.dtype, or Python dtype, default None
```

Comment:
@jreback : huh?  Read what you wrote <a href="https://github.com/pydata/pandas/pull/13425#issuecomment-228307584">here</a>. That was not made clear **at all** to me beforehand.  Why didn't you provide this explanation earlier?

Also, what do you mean by "typical" usage?  Where else is such behaviour implemented in the codebase?